client/tx/utils.go

```diff
@@ -0,0 +1,173 @@
+package tx
+
+import (
+	"encoding/hex"
+	"errors"
+	"strings"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+)
+
+// SearchTxs performs a search for transactions for a given set of tags via
+// Tendermint RPC. It returns a slice of Info object containing txs and metadata.
+// An error is returned if the query fails.
+func SearchTxs(cliCtx context.CLIContext, cdc *codec.Codec, tags []string, page, limit int) ([]sdk.TxResponse, error) {
```

Comment:
It'd be nice to have a test for this function

Reply:
I agree, but this is tested via the CLI tests mostly. In order to test this, we'd have to have a node configured and running.

Reply:
Fair enough